### PR TITLE
added doc about 'tree_leaf_edit' and 'tree_full_edit'

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A |
| --- | --- |
| Fixed tickets | none |
| License | MIT |
#### What's in this PR?

This PR add the doc about 'tree_leaf_edit' and 'tree_full_edit'.

### CODE REVIEW

File: book/creating-a-basic-website/setup-a-webspace.rst [hunk 1]
```diff
@@ -142,12 +142,21 @@ developer can retrieve the navigation for a given context by using some
...
 
-Resource-Locator
-----------------
+Resource-Locator (optional)
+---------------------------
 
 The `strategy` for the `resource-locator` influences the design of the URLs for
-the content. Currently there is only the `tree`-option available resulting in
-exposing the entire content tree in the URL.
+the content. Default value is `tree_leaf_edit` which means that the
```
Comment (danrot) [inline book/creating-a-basic-website/setup-a-webspace.rst]: `tree_leaf_edit, which means`

File: book/creating-a-basic-website/setup-a-webspace.rst [hunk 2]
```diff
@@ -142,12 +142,21 @@ developer can retrieve the navigation for a given context by using some
...
+`resource-locator` will be generated for the whole tree but only the last part
```
Comment (danrot) [inline book/creating-a-basic-website/setup-a-webspace.rst]: `whole tree, but only`

File: book/creating-a-basic-website/setup-a-webspace.rst [hunk 3]
```diff
@@ -142,12 +142,21 @@ developer can retrieve the navigation for a given context by using some
...
+will be editable.
+
+Currently there is only one alternative `tree_full_edit` which uses also
```
Comment (danrot) [inline book/creating-a-basic-website/setup-a-webspace.rst]: `tree_full_edit, which also generates the whole tree, but lets you`

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**